sdk/purview/purview-account-rest/samples/v1/javascript/README.md

```diff
@@ -0,0 +1,63 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+urlFragment: purview-account-javascript
+disableDocsMs: true
+---
+
+# Azure Purview Scanning rest client library samples for JavaScript
+
+These sample programs show how to use the JavaScript client libraries for Azure Purview Scanning rest in some common scenarios.
+
+| **File Name**           | **Description**                      |
+| ----------------------- | ------------------------------------ |
+| [typedefs.js][typedefs] | gets a list of typedefs for entities |
+
+## Prerequisites
+
+The sample programs are compatible with Node.js >=12.0.0.
+
+You need [an Azure subscription][freesub] and the following Azure resources to run these sample programs:
+
+- [Azure Cognitive Services instance][createinstance_azurecognitiveservicesinstance]
```

Comment:
do we really need a cognitive services instance for purview?

Reply:
updated.